tests/core/test_datastructures.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import json
+
+from pydantic.alias_generators import (
+    to_camel,
+)
+
+from tests.core._test_utils import (
+    TEST_SIGNED_AUTHORIZATION,
+    PydanticTestClass,
+)
+
+
+@pytest.mark.parametrize(
+    "pydantic_model,expected",
+    (
+        (
+            TEST_SIGNED_AUTHORIZATION,
+            {
```

Comment:
Nit: Expected data could be pulled into _test_utils

Reply:
def worth it as it's already used in two places 👍🏼 ... updated